test/utility/test_util.i

```diff
@@ -98,7 +98,8 @@ testutil_clean_work_dir(char *dir)
 
 	/* 10 bytes for the Windows rd command */
 	inputSize = strlen(dir) + sizeof(RM_COMMAND);
-	buffer = (char*) malloc (inputSize);
```

Comment:
The spaces are wrong on these lines. There should not be a space after the case, before the function parameters, or after the NULL.